npm-shrinkwrap.json

```diff
@@ -1636,11 +1683,703 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "dev": true
     },
+    "fsevents": {
```

Comment:
well, it won't crash, but the behavior of a shrinkwrap in case of optional OS dependencies is totally odd. When I start building more stuff, the dependency might disappear from the shrinkwrap and when you change a dependency, it might appear again. Just have a look at [this](https://github.com/angular/angular/blob/e86573bac8a87787e415d1615b09f7e3d30ecb30/npm-shrinkwrap.readme.md) for further details.
However, I think that we shouldn't waste time with that now and wait until "real" problems appear to start solving this problem.